src/ask.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const readline = require('readline');
+
+const terminal = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+function getInput(message) {
+  return new Promise(resolve => {
+    terminal.question(message, input => {
+      resolve(input);
+    });
+  });
+}
+
+function printOut(message) {
+  global.console.log(message);
```

Comment:
It is not correct, because this function works incorrectly with terminal write
![image](https://user-images.githubusercontent.com/104682987/236393199-00dd31da-2a62-4346-a25d-90b3ec4c98bc.png)

Reply:
with global.console.log => 
![image](https://user-images.githubusercontent.com/104682987/236393438-192b3648-3e77-43f0-9df4-b76df8f04f5f.png)